packages/Search/src/Search.js

```diff
@@ -0,0 +1,138 @@
+import React from "react";
+import PropTypes from "prop-types";
+import ListBox from "@paprika/list-box";
+import SearchIcon from "@paprika/icon/lib/Search";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import { filter } from "@paprika/list-box/lib/helpers/filter";
+import renderTrigger from "./renderTrigger";
+import useTrigger from "./useTrigger";
+
+const propTypes = {
+  children: PropTypes.node.isRequired,
+  onChangeSearch: PropTypes.func,
+  onSelected: PropTypes.func,
+};
+
+const defaultProps = {
+  onChangeSearch: () => {},
+  onSelected: () => {},
+};
+export default function Search(props) {
+  const refSelected = React.useRef(null);
+  const { children, onChangeSearch, onSelected, ...moreProps } = props;
+  const { t } = useI18n();
+  const {
+    inputValue,
+    onBlurInput,
+    onBlurTrigger,
+    onChangeInput,
+    onClickTrigger,
+    onKeyDownTrigger,
+    refInput,
+    refListBoxReducer,
+    resetValue,
+    setInputValue,
+  } = useTrigger();
+
+  const countOptions = React.useMemo(() => {
+    let count = 0;
+    React.Children.forEach(children, child => {
+      if (child.type.displayName === "ListBox.Option") count += 1;
+    });
+
+    return count;
+  }, [children]);
+
+  const refDivRoot = React.useRef(null);
+  /* eslint-disable react/prop-types */
+  const size =
+    typeof props.size !== "undefined" && Object.keys(ListBox.types.size).includes(props.size.toUpperCase())
+      ? props.size
+      : ListBox.types.size.MEDIUM;
+  /* eslint-enable react/prop-types */
+
+  function processSelected(value) {
+    const dispatch = refListBoxReducer.current.dispatch;
+    const types = refListBoxReducer.current.types;
+
+    const actions = {
+      open() {
+        dispatch({ type: types.openPopover });
+      },
+      close() {
+        dispatch({ type: types.closePopover });
+      },
+      cleanInput() {
+        resetValue(); // input search value
+      },
+      setInput(value) {
+        setInputValue(value);
+      },
+    };
+    refSelected.current = null; // selected ref value
+    onSelected(value, actions);
+  }
+
+  function handleChange(index, options, attributes) {
+    const { eventType } = attributes;
+
+    if (eventType.includes("click")) {
```

Comment:
Only `click` event and trigger a select? keyboard `enter` is not allowed?

Reply:
yes is allowed under the submit event on the form element.